README.md

```diff
@@ -1,26 +1,40 @@
-# subgraph-cosmos-example
+# Cosmos Data Dump Example Subgraph
 
-## Setup
+This example subgraph decodes full blocks using all available proto definitions and saves them to the subgraph store. The following steps will setup a local firehose instance and deploy the example subgraph. This can also be done using a firehose instance deployed to the hosted service.
```

Comment:
> This example subgraph

I would drop "example" here.

> decodes full blocks using all available proto definitions

This is a bit misleading – the data is already decoded.

> This can also be done using a firehose instance deployed to the hosted service.

This subgraph is for internal use only, we don't need to mention Hosted Service here.